forge/test/mlir/test_ops_tf.py

```diff
@@ -118,7 +118,6 @@ def call(self, x):
     if (
         test_param
         in [
-            (1, 64, 16, 115, 115, 4, 4, 1, 1),
             (1, 64, 64, 56, 56, 3, 3, 1, 1),
             (1, 128, 128, 56, 56, 3, 3, 2, 2),
             (1, 64, 64, 8, 8, 3, 3, 1, 1),
```

Comment:
Can you please remove the case `(1, 256, 256, 7, 7, 3, 3, 1, 1)` from the list and the below xfail (for n300)?

It should pass with latest metal 😄

Reply:
Running CI to confirm if it's working :))
- https://github.com/tenstorrent/tt-forge-fe/actions/runs/12047015293/job/33588693883